patch/kernel/rockchip64-dev/general-rockchip-overlays.patch

```diff
@@ -133,6 +134,10 @@ index e69de29..9512445 100644
 +Notice: UART4 cannot be activated together with SPI1 - they share the sam pins.
 +Enabling this overlay disables SPI1.
 +
++### usb-c-host
```

Comment:
Maybe we should name it in a more generic manner eg. `dwc3-0-host`?
This is type C port for NanoPC T4 but the overlay could also be used with ROCK Pi 4 for which `usb-c-host` would be misleading ;-)

Reply:
I don't really have an objection to that. Do you want me to recreate the Pull Request or is this something you can change during integration?

Reply:
You can make an additional commit to adjust it.
If not I can do it.

Reply:
Done.